makefile

```diff
@@ -0,0 +1,14 @@
+default:
+	cat makefile
```

Comment:
makefiles, like python are space sensitive, and although this will work, it is a little easier on the eyes if you separate your commands with at least one empty line.

Reply:
no sweat, updates committed